code/datums/mutations/_combined.dm

```diff
@@ -32,3 +32,6 @@
 /datum/generecipe/antiglow
 	required = "/datum/mutation/human/glow; /datum/mutation/human/void"
 	result = ANTIGLOWY
+/datum/generecipe/superhearing
+	required = "/datum/mutation/human/antenna; /datum/mutation/human/radioactive"
```

Comment:
I'd suggest that instead of doing this, you make this a species-specific mutation for moths. Look to Fire Breath for relevant code (it's exclusively natural to lizards), it's actually really easy to do

Reply:
I did that, didnt I? wait let me see